src/floating-focus.spec.js

```diff
@@ -88,22 +85,72 @@ describe('Floating focus', () => {
 		expect(floatingFocus.disableFloatingFocus).toHaveBeenCalled();
 	});
 
-	it('Should only reposition if a target and \'floater\' was set, when scrolling or resizing', async () => {
+	it('Scrolling should not reposition by default.', async () => {
```

Comment:
I'm always a bit pedantic about the name of tests with `it` hehe... Usually I'd actually prefer something like `it('does not reposition the floater by default'` (so without "should") but let's stick to the convention of the rest for the tests.
```suggestion
	it('Should not reposition the floater by default', async () => {
```